sonic-chassisd/setup.py

```diff
@@ -15,6 +15,7 @@
     ],
     scripts=[
         'scripts/chassisd',
+        'scripts/chassisshow'
```

Comment:
should we call it chassis_show instead to simpler read? i reads it as chassis how :-)

Reply:
Yes, that would be better. I did look when initially naming it and most SONiC native scripts do not have a underscore, however if you think it will be fine then I would definitely want to add one.